### ISSUE TITLE
inFamous [NPUA80480] crashes on Vulkan after #5565

### ISSUE THREAD

**Issue by RainbowCookie32** [RPCS3/rpcs3#5600]
After #5565 inFamous crashes consistently at basically the same spot (sometimes it runs for like half a second more or less, but it always crashes). Works fine on #5579, and the same issue is present in current master. The fatal is 

```
{RSX [0x0555028]} RSX: class std::runtime_error thrown: [attrib buffer] Working buffer not big enough, buffer_length=402653184 allocated=402317824 requested=211658 guard=4194304 largest_pool=536576
(in file c:\projects\rpcs3\rpcs3\emu\rsx\common\ring_buffer_helper.h:78)
```

Screenshot from the place where it crashes: 
![untitled](https://user-images.githubusercontent.com/16805474/51792957-c4614c80-2197-11e9-8c1f-6896aad5f5c5.png)

I tried OpenGL and the game seems to be working fine with it.

My Specs:
Ryzen 3 1200, GTX 1050, 8GB RAM, Windows 10.

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/2799511/RPCS3.log.gz)

**Comment by creeperjedi** [comment]
I get the same results when using Vulkan, but when I use OpenGL I get issues with exploding vertices during car explosions (and other random things that I can't replicate consistently). The only way to avoid the exploding vertices is to look away from the car explosion, after which you can look back without any problem.
Logs:
[Infamous Log - 0.0.5-7755.zip](https://github.com/RPCS3/rpcs3/files/2799518/Infamous.Log.-.0.0.5-7755.zip)
[Infamous Log.gz - 0.0.5-7755.zip](https://github.com/RPCS3/rpcs3/files/2799519/Infamous.Log.gz.-.0.0.5-7755.zip)
And a RSX capture of the exploding vertices (I had to split it up because github only supports 10MB files):
[Infamous RSX Capture (OpenGL) - 0.0.5-7755.7z.zip](https://github.com/RPCS3/rpcs3/files/2799528/Infamous.RSX.Capture.OpenGL.-.0.0.5-7755.7z.zip)
[Infamous RSX Capture (OpenGL) - 0.0.5-7755.7z (2).zip](https://github.com/RPCS3/rpcs3/files/2799529/Infamous.RSX.Capture.OpenGL.-.0.0.5-7755.7z.2.zip)
[Infamous RSX Capture (OpenGL) - 0.0.5-7755.7z (3).zip](https://github.com/RPCS3/rpcs3/files/2799530/Infamous.RSX.Capture.OpenGL.-.0.0.5-7755.7z.3.zip)
[Infamous RSX Capture (OpenGL) - 0.0.5-7755.7z (4).zip](https://github.com/RPCS3/rpcs3/files/2799531/Infamous.RSX.Capture.OpenGL.-.0.0.5-7755.7z.4.zip)

**Comment by RainbowCookie32** [comment]
Indeed, I was just going to edit the issue to add that. By the way, you can upload the capture somewhere else, like MEGA and post the link here

**Comment by RainbowCookie32** [comment]
Both issues are still present on current master (0.0.6-7909-a254a203)

**Comment by kd-11** [comment]
Fixed by https://github.com/RPCS3/rpcs3/pull/5988
